backend/test_appdata/test-package_function_references_work.json

```diff
@@ -9,7 +9,7 @@
       "ast": [
         "EFnCall",
         503886666,
-        "test_admin/stdlib/Test::one_v0",
+        "test_admin/stdlib/Test::one_v1",
```

Comment:
Maybe something to pull out into a separate issue:
without changing this (and other corresponding locations) to be a v1 rather than a v0, the _v0 portion of the function name was stripped at the call site. Seemingly this is an OCaml interop issue, but couldn't pin down the specific issue.

Reply:
Yeah, that sounds right. This fix is fine - it's an awkward thing to fix in both versions